src/jit/compiler.cpp

```diff
@@ -7202,7 +7225,8 @@ void JitTimer::PrintCsvHeader()
 
     CritSecHolder csvLock(s_csvLock);
 
-    if (_waccess(jitTimeLogCsv, 0) == -1)
+    FILE* fp = _wfopen(jitTimeLogCsv, W("r"));
+    if (fp == nullptr)
```

Comment:
I didn't realize that this conflicts with parjong's change; I'll retest on Windows and Linux (and OS X in a bit, I don't have that set up yet). This snippet is a wee bit "subtle", if you get my drift, and I think that we need to preserve the existing semantics (e.g. not just append since that might mix entries from multiple runs, as the comments explain).